pkg/uroot/uroot.go

```diff
@@ -26,6 +27,117 @@ var (
 	}
 )
 
+// Opts are the arguments to CreateInitramfs.
+type Opts struct {
+	// Env is the build environment (OS, arch, etc).
+	Env golang.Environ
+
+	// Builder is the build format.
+	Builder Build
+
+	// Archiver is the initramfs archival format.
+	Archiver Archiver
+
+	// Packages are the Go packages to add to the archive.
+	//
+	// Currently allowed formats:
+	//   Go package imports; e.g. github.com/u-root/u-root/cmds/ls
+	//   Paths to Go package directories; e.g. $GOPATH/src/github.com/u-root/u-root/cmds/*
```

Comment:
btw, this says *, but when I tried it, it did not work. Should it?

Reply:
It shouldn't, removed that.

Reply:
I'll never agree with taking the Glob out but you know that already :-)
it just seems to push the problem onto all the potential users of this package.